gcp/appengine/blog/content/posts/announcing-api-queries-for-more-linux-distros/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "API Queries for More Linux Distributions"
+date: 2024-10-22T00:00:00Z
+draft: false
+author: Holly Gong
+---
+
+We're excited to announce that OSV.dev's API now allows you to query all our supported Linux distributions!
+
+2024 has seen significant adoption of the OSV Schema from prominent Linux distributions such as [Ubuntu](https://openssf.org/blog/2024/06/11/ubuntu-security-notices-now-available-in-osv/), [Chainguard/Wolfi](https://openssf.org/blog/2024/07/03/chainguard-enhances-security-with-osv-advisory-feed/), and [SUSE/openSUSE](https://www.suse.com/support/update/). In particular, Ubuntu provides us with both [Ubuntu Security Notices](https://ubuntu.com/security/notices) (identified as `USN-`) and [Ubuntu CVE Tracker](https://ubuntu.com/security/cves) (identified as `UBUNTU-CVE-`) to cover fixed and unfixed vulnerabilities. Chainguard/Wolfi and SUSE/openSUSE have also recently adopted the OSV Schema. This increased community adoption allows us to expand our Linux distribution vulnerability coverage significantly.
+
+Although we had expanded our coverage of Linux distributions, we didn't support API queries for many of them due to limitations in our query implementation. Specifically, the OSV.dev API relied solely on enumerated affected versions for package version queries, requiring version enumeration functions to be implemented for each ecosystem. This approach was difficult to scale and limited our API queries to only Debian, Ubuntu, and Alpine, and was a barrier to utilizing the data already available.
```

Comment:
I think it's worth expanding on this about how it was also a barrier to new ecosystems onboarding with OSV.dev in a way that made the data actually usable?

Reply:
Expanded a bit more on this.